packages/orchestration/src/examples/swapExample.contract.js

```diff
@@ -79,16 +80,20 @@ export const start = async (zcf, privateArgs, baggage) => {
     timerService,
     chainHub,
   );
+  const { accountsStorageNode } = await provideAll(baggage, {
+    accountsStorageNode: () => E(storageNode).makeChildNode('accounts'),
+  });
```

Comment:
What's the thought behind the parent of orchestration being `accounts`? It will need to put other info in there, won't it?

I won't block on this since the code is still experimental and we have #9066 ahead of us. But consider `makeChildNode('orchestration')` since that's the scope it's being handed to.

```suggestion
  const { orchestrationStorageNode } = await provideAll(baggage, {
    orchestrationStorageNode: () => E(storageNode).makeChildNode('orchestration'),
  });
```

We should try to be consistent across contracts so that we can have a helper that does all this setup in one call from `start`.

Reply:
We make separate nodes for `stakeAtom` and `stakeOsmo` in the `start-*.js` scripts, and each has an `accounts` child node. Presumably other info or child nodes can be written to the `stakeAtom` and `stakeOsmo` paths.

This seems to be in line with "delegates vstorage publishing responsibilities to guest contracts" in the PR description.

Agree it may make sense to globally publish something for all accounts, unless explicitly opted out, and that's where I think an `orchestration` node makes sense.